pyconcz/proposals/models.py

```diff
@@ -118,10 +119,11 @@ class Talk(EntryBase):
         upload_to='proposals/pyconcz2018/talks/',
```

Comment:
Careful with the path here. it says 2018.

Reply:
Fixed